test/runtime/index.ts

```diff
@@ -255,6 +255,7 @@ describe('runtime', () => {
 					flush();
 
 					if (config.after_test) config.after_test();
+					done();
```

Comment:
Maybe we could chain a `.finally` call last that calls `done()`? And also for line 247 too

Reply:
`.finally` is added on ES2018 but `compilerOptions.lib` of `tsconfig` is `es2017`.
Therefore I used polyfill.
Can I add `es2018.promise` on `test/tsconfig.json`?

If I can use `.finally`, calling `done()` can be very simple and readable.

Reply:
Good point. I got confused with the try..catch..finally syntax, but looks like you're right, `.finally` isn't available in node 8.